documentation/docs/05-load.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Loading data
+---
+
+A [`+page.svelte`](/docs/routing#page-page-svelte) or [`+layout.svelte`](/docs/routing#layout-layout-svelte) gets its `data` from a `load` function.
+
+If the `load` function is defined in `+page.js` or `+layout.js` it will run both on the server and in the browser. If it's instead defined in `+page.server.js` or `+layout.server.js` it will only run on the server, in which case it can (for example) make database calls and access private [environment variables](/docs/modules#$env-static-private), but can only return data that can be serialized as JSON.
+
+```js
+/// file: src/routes/+page.js
+// @filename: $types.d.ts
+export type PageLoad = import('@sveltejs/kit').Load<{}>;
+
+// @filename: index.js
+// ---cut---
+/** @type {import('./$types').PageLoad} */
+export function load(event) {
+	return {
+		some: 'data'
+	};
+}
+```
+
+### Input properties
+
+The argument to a `load` function is a `LoadEvent` or (for server-only `load` functions) a `ServerLoadEvent`, which inherits `clientAddress`, `locals`, `platform` and `request` from `RequestEvent`. All events have the following properties:
```

Comment:
```suggestion
The argument to a `load` function is a `LoadEvent` or (for server-only `load` functions a `ServerLoadEvent`), which inherits `clientAddress`, `locals`, `platform` and `request` from `RequestEvent`. All events have the following properties:
```

Reply:
not a typo! The way to tell if parens are in the right place is to remove the clause entirely and see if it makes sense:

> The argument to a `load` function is a `LoadEvent` or a `ServerLoadEvent`, which inherits `clientAddress`, `locals`, `platform` and `request` from `RequestEvent`.

vs

> The argument to a `load` function is a `LoadEvent` or, which inherits `clientAddress`, `locals`, `platform` and `request` from `RequestEvent`.

An alternative would be this, which is technically redundant but might make it clearer that `clientAddress` etc only apply to `ServerLoadEvent`:

```suggestion
The argument to a `load` function is a `LoadEvent` (or, for server-only `load` functions, a `ServerLoadEvent` which inherits `clientAddress`, `locals`, `platform` and `request` from `RequestEvent`). All events have the following properties:
```